internal/gps/pkgtree/reachmap.go

```diff
@@ -20,44 +20,37 @@ type ReachMap map[string]struct {
 	Internal, External []string
 }
 
-// FlattenAll flattens a reachmap into a sorted, deduplicated list of all the
-// external imports named by its contained packages.
-//
-// If stdlib is false, then stdlib imports are excluded from the result.
-func (rm ReachMap) FlattenAll(stdlib bool) []string {
```

Comment:
Hmm...I guess we're not using, let alone testing, this right now. That's not great, but I think it's something we may end up coming back to want in the future. Even if we do drop it, I'd rather not redesign the API in a way that would make it awkwardly dissimilar to its siblings.